test/newline_test.rb

```diff
@@ -38,15 +38,15 @@ def visit(node)
         expected.delete_at actual.index(62)
       elsif relative == "lib/yarp/lex_compat.rb"
         # extra flag for: dedent_next =\n  ((token.event: due to bytecode order
-        actual.delete(520)
+        actual.delete(521)
         # different line for: token =\n  case event: due to bytecode order
-        actual.delete(577)
-        expected.delete(578)
+        actual.delete(578)
+        expected.delete(579)
         # extra flag for: lex_state =\n  if RIPPER: due to bytecode order
-        actual.delete(610)
+        actual.delete(611)
         # extra flag for: (token[2].start_with?("\#$") || token[2].start_with?("\#@"))
         # unclear when ParenthesesNode should allow a second flag on the same line or not
-        actual.delete(737)
+        actual.delete(738)
```

Comment:
These are rather annoying to update, not sure if there is a good solution.
FWIW for serialize.rb I changed it to find the line automatically: https://github.com/ruby/yarp/commit/5de581923103911d7eba1019528a5827ecf3a980#diff-bcb8a82557341f407dbe084da75234f3139581f75d4c23d0c0e272af5aef12aa